src/unstable-temp/DownloadImageModal/FormModel.tsx

```diff
@@ -173,6 +176,29 @@ const FormControl = ({ onModelChange, options, model }: FormControlProps) => {
 						</Box>
 					)}
 
+					{options.type === 'datetime-local' && (
+						<Input
+							mb={2}
+							type="datetime-local"
+							value={
```

Comment:
```suggestion
							name={options.name}
							value={
```
W/o this ^ no expiry date is set when downloading an image.